tests/cpp/cell/test_g2r_copy.cu

```diff
@@ -9,30 +10,54 @@ namespace tiledcuda {
 
 using namespace cell;
 
-template <typename Element, tl::Layout type, size_t height, size_t width>
-__global__ void copy_g2r(Element* src) {
-    using SrcTile = GlobalTile<Element, tl::RowMajor<16 * height, 16 * width>>;
-    using DstTile = RegTile<RegTile<Element, tl::RowMajor<2, 4>>,
-                            tl::RowMajor<height, width>>;
+template <typename Element, tl::Layout type, typename WarpLayout,
+          typename RegLayout, const copy::WarpReuse kMode, const size_t kHeight,
+          size_t kWidth>
+__global__ void copy_g2r_row_major(Element* src) {
+    using SrcTile =
+        GlobalTile<Element, tl::RowMajor<16 * kHeight, 16 * kWidth>>;
+    using DstTile = RegTile<BaseTileRowMajor<Element>, RegLayout>;
     SrcTile src_tile(src);
     DstTile dst_tile;
 
-    cell::copy::GlobalToRegLoader<SrcTile, DstTile, type> loader;
+    cell::copy::GlobalToRegLoader<SrcTile, DstTile, WarpLayout, kMode> loader;
     loader(src_tile, dst_tile);
     __syncthreads();
 
     if (threadIdx.x == 0) {
-        for (int i = 0; i < height; ++i) {
-            for (int j = 0; j < width; ++j) {
+        printf("thread 0:\n");
+        for (int i = 0; i < DstTile::kRows; ++i) {
+            for (int j = 0; j < DstTile::kCols; ++j) {
+                dst_tile(i, j).dump_value();
+            }
+        }
+    }
```

Comment:
you can try directly, if this https://github.com/TiledTensor/TiledCUDA/blob/a84953153809b62aa7fc78f003abeb2474d3c57e/include/util/print.hpp#L43 works properly.

```cpp
dst_tile.dump_value();
```

Reply:
fix.